o8g/definition.xml

```diff
@@ -65,27 +65,27 @@
     background="table/background.png"
     backgroundStyle="uniformToFill"
     ordered="False">
-    <groupaction menu="Set up" execute="setup" showIf="setup_required"/>
-    <groupaction menu="Pass" execute="pass_action" showIf="setup_not_required"/>
-    <groupaction menu="Give honor" execute="give_honor" showIf="setup_not_required"/>
-    <groupaction menu="Declare conflict" execute="declare_conflict" showIf="setup_not_required"/>
-    <groupaction menu="Claim ring" execute="claim_ring" showIf="setup_not_required"/>
-    <groupaction menu="End turn" execute="end_turn" showIf="setup_not_required"/>
-    <groupaction menu="Flip coin" execute="flip_coin" showIf="setup_not_required"/>
+    <groupaction menu="Set up" execute="setup" showIf="setup_required" shortcut="CTRL+SHIFT+S" />
```

Comment:
well there is some actions that share letters, so i had to find something else.
Some are also coming from conquest yeah, like ALT+D for pass, was the shortcuts for "i'm done"